tests/album-create.test.js

```diff
@@ -0,0 +1,48 @@
+// tests/album-create.js
+const { expect } = require('chai');
+const request = require('supertest');
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+describe('create album', () => {
+  //first create a new artist to use in album table
+  let artist;
+  let db;
+  beforeEach(async () => {
+    db = await getDb();
+    await db.query('INSERT INTO Artist (name, genre) VALUES (?,?)', [
+      'Tame Impala',
+      'Rock',
+    ]);
+    [artist] = await db.query(`SELECT * FROM Artist`);
+  });
+
+  afterEach(async () => {
+    //delete all from album table after testing
+    await db.query('DELETE FROM Album');
+    await db.close();
+  });
+
+  describe('/artist/:id/album', () => {
+    describe('POST', () => {
+      it('creates a new album in the database', async () => {
```

Comment:
Could another test be added to check for error cases (such as trying to add an album to an artist that doesn't exist)?